networking/bgp.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: Configure BGP peering
-description: Configure BGP peering for public cloud and on-premises deployments with full mesh, node-specific peering, ToR and/or Calico route reflectors.
+description: Configure BGP peering with full mesh, node-specific peering, ToR, and/or Calico route reflectors.
 canonical_url: '/networking/bgp'
 ---
 
 ### Big picture
 
-Configure border gateway protocol (BGP) peering with network infrastructure to distribute routing information.
+Configure BGP (Border Gateway Protocol) between Calico nodes or peering with network infrastructure to distribute routing information.
 
 ### Value
 
-{{site.prodname}} nodes can exchange routing information over BGP to enable reachability for {{site.prodname}} networked workloads without the need for encapsulation.
+{{site.prodname}} nodes can exchange routing information over BGP to enable reachability for {{site.prodname}} networked workloads (Kubernetes pods or OpenStack VMs). In an on-prem deployment this allows you to make your workloads first class citizen across the rest of your network. In public cloud deployments it provides an efficient way of distributing routing information within your Cluster, often used in conjunction with IPIP overlay or cross-subnet modes.
```

Comment:
In an on-premises deployment this allows you to make your workloads first-class citizen across the rest of your network. In public cloud deployments, it provides an efficient way of distributing routing information within your cluster, and is often used in conjunction with IPIP overlay or cross-subnet modes.